dev/20-componentconfig-gardenlet.yaml

```diff
@@ -0,0 +1,116 @@
+---
+apiVersion: gardenlet.config.gardener.cloud/v1alpha1
```

Comment:
How does this file was included in this PR when dev directory is excluded here ? https://github.com/gardener/gardener/blob/1ea177fa7ead2be9122368affbac2e634f08a175/.gitignore#L6

Reply:
Removed with https://github.com/gardener/gardener/pull/3789.